### PR TITLE
Add team variable to getDomainUrl()

### PR DESCRIPTION
**Pull request by jonohewitt**
## Description

Small update to use a dynamic `team` variable instead of the hardcoded extension author's username `'destiner'` that resulted in broken URLs for other users. It seems like this would have been @Destiner's original intention but it just slipped through the net before publishing.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/netlify/package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "description": "View your Netlify sites, domains, and team members",
   "icon": "icon.png",
   "author": "destiner",
+  "contributors": ["jonohewitt"],
```
Comment (danielsequeira) [inline extensions/netlify/package.json:8]: @jonohewitt we don't really set the contributors property in our extensions

**Comment by danielsequeira** [comment]
@jonohewitt do you think you could add a couple of screenshots of this extension in the metadata folder, by following this guide: https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots

File: extensions/netlify/package.json (hunk 1 above)
Comment (jonohewitt) [reply extensions/netlify/package.json:8]: Okay no problem! Did I misunderstand something here or does it need updating?

https://github.com/raycast/extensions/blob/b946ec2ca121f9ab95dcf6850728ecb6380d01ba/basics/contribute-to-an-extension.md?plain=1#L23

**Comment by jonohewitt** [comment]
> @jonohewitt do you think you could add a couple of screenshots of this extension in the metadata folder, by following this guide: https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots

@danielsequeira Sorry, it seems like any meaningful screenshots would show my personal Netlify info which I'd prefer not to use.

**Comment by danielsequeira** [comment]
> > @jonohewitt do you think you could add a couple of screenshots of this extension in the metadata folder, by following this guide: https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots
> 
> @danielsequeira Sorry, it seems like any meaningful screenshots would show my personal Netlify info which I'd prefer not to use.

No you didn't, I did. Forgot that we actually ask for people to do this when contributing to extensions.

That is a shame, however understandable.

**Review by danielsequeira — APPROVED** [review]

**Comment by danielsequeira** [comment]
@jonohewitt would you mind adding a changelog entry mentioned your fix?

**Comment by jonohewitt** [comment]
Ah sorry I hadn't noticed the formatting was wrong! Thanks for reviewing @danielsequeira

**Pull request merged**